misc/test/examples_test.go

```diff
@@ -67,6 +67,32 @@ func TestExamples(t *testing.T) {
 				expectHelloWorld(t, stack.Outputs["publicIP"])
 			},
 		}),
+		base.With(integration.ProgramTestOptions{
+			Dir:       path.Join(cwd, "..", "..", "azure-ts-functions"),
```

Comment:
I already submitted test coverage for this one yesterday in https://github.com/pulumi/examples/pull/139 based on the work we did in the AM.